Formula/gdrive.rb

```diff
@@ -0,0 +1,22 @@
+class Gdrive < Formula
```

Comment:
This file should be the same as one we use on tap:
 https://github.com/paulz/homebrew-gdrive/blob/master/gdrive.rb

See https://github.com/paulz/homebrew-gdrive#usage
```
brew tap paulz/gdrive
brew install gdrive
```